release.md

```diff
@@ -105,14 +105,16 @@ Fields to update in `recipe/meta.yaml` of feedstock repo:
 #### Update release notes
 1. Replace "Future Release" in `docs/source/release_notes.rst` with the current date
     ```
-    **v0.13.3 Feb 28, 2020**
+    v0.13.3 Feb 28, 2020
+    ====================
     ```
 2. Remove any unused sections for this release (e.g. Fixes, Testing Changes)
 3. Add yourself to the list of contributors to this release and put the contributors in alphabetical order
 4. The release PR does not need to be mentioned in the list of changes
 5. Add a commented out "Future Release" section with all of the release notes sections above the current section
     ```
-    .. **Future Release**
+    .. Future Release
+    ==============
```

Comment:
got a warning when I added this template to the release notes, adding a 2 space indent to this line fixed it

Reply:
Ah, ok. Wasn't real sure how to handle this one cleanly. Added spaces.